scripts/debian/builder-helpers.sh

```diff
@@ -518,7 +532,7 @@ build_daemon_mainnet_legacy_deb() {
 # Builds mina-devnet-legacy package with legacy testnet binary
 #
 # Output: mina-devnet-legacy_${MINA_DEB_VERSION}_${ARCHITECTURE}.deb
-# Dependencies: ${SHARED_DEPS}${DAEMON_DEPS}
+# Dependencies: ${SHARED_DEPS}${DAEMON_DEBS}
```

Comment:
Various `_DEP` suffix is replaced as `_DEB`, why is that happening in this script? I thought DEP means dependency right?